src/components/common/AppHeader.vue

```diff
@@ -72,26 +75,35 @@
 
 <script>
 import config from "src/../config"
-import { mapState } from "vuex"
+import { mapState, mapGetters } from "vuex"
 import noScroll from "no-scroll"
 import AppMenu from "common/AppMenu"
+import { NotificationAdded } from "src/gql"
 export default {
   name: `app-header`,
   components: { AppMenu },
   data: () => ({
     open: false,
     desktop: false,
     isMobileApp: config.mobileApp,
+    allSessionAddresses: [],
+    notificationCounter: 0, // needs to be fetched from store
   }),
   computed: {
     ...mapState([`session`, "connection"]),
+    ...mapGetters([`networks`]),
     networkSlug() {
       return this.connection.networkSlug
     },
   },
-  mounted() {
+  mounted: async function () {
     this.watchWindowSize()
     window.onresize = this.watchWindowSize
+    const networkIds = this.networks.map((network) => network.id)
+    this.allSessionAddresses = await this.$store.dispatch(
```

Comment:
needs to be dynamic as it changes on every sign in?

Reply:
dynamic how do you mean?

Reply:
if you sign in to a new address, this will not be reflected here

Reply:
ping

Reply:
but this doesn't matter, with which account you are signed in. Sessions in localStorage are the same

Reply:
when you sign in to new network that you didn't have a session for before `allSessionAddresses` will not update and the user will not subscribe to the new session address